packages/expo-firebase-analytics/src/Analytics.ts

```diff
@@ -196,12 +222,11 @@ export function setClientId(clientId: string): void {
 }
 
 /**
- * Enables or disabled debug mode on the Expo client, so events can
+ * __Expo Go Only.__ Enables or disabled debug mode on the Expo client, so events can
```

Comment:
Wonder if maybe we should add a thing like `@runtime` that we can use with `Expo Go/Custom` or maybe some other indication like the Expo Go version.

Reply:
That's a good idea, I have the platform specific annotations (`@ios`, `@android`, `@web`) on the TODO list already, but I wanted to wait for the docs redesign, which will rewrite a lot of UI code and should include a component which can be easily used as label in docs. 

So currently I just stick to plain text information, which can been seen in the docs before I have started the work on `APISection`. 

In this specific case the docs have this annotation already, but the code comment did not:
* https://github.com/expo/expo/pull/14302/files#diff-2f1f696c927c4e572b9cf5a22dd9ebb3b7bfe677c2cf10ffcaa0bbed7be78eb5L326